README.md

```diff
@@ -106,40 +106,62 @@ bq mk -t prod.analytics bigquery/analytics_schema.json
+        "urls": [
+          "turn:hostnameForYourTurnServer:19305?transport=udp",
+          "turn:hostnameForYourTurnServer:19305?transport=tcp"
+        ],
+        "username": "TurnServerUsername",
+        "credential": "TurnServerCredentials"
+      },
+      {
+        "urls": [
+          "stun:hostnameForYourStunServer:19302"
+        ]
+      }
     ]
-  }
-]
-```
-7\. **(Only consider this if you skipped step 5 and 6)** AppRTC by default uses an ICE server provider to get TURN servers. Previously we used a [compute engine on demand service](https://github.com/juberti/computeengineondemand) (it created TURN server instances on demand in a region near the connecting users and stored them in shared memory) and web server with a REST API described in [draft-uberti-rtcweb-turn-rest-00](http://tools.ietf.org/html/draft-uberti-rtcweb-turn-rest-00). This has now been replaced with a Google service. It's similar from an AppRTC perspective but with a different [response format](https://github.com/webrtc/apprtc/blob/master/src/web_app/js/util.js#L77). You would have to setup this yourself or hard code your TURN servers in step 6.
+    ```
+
+* **Else if using ICE Server provider [1]**
+  * Change `ICE_SERVER_BASE_URL` to your ICE server provider host.
+  * Change `ICE_SERVER_URL_TEMPLATE` to a path or empty string depending if your ICE server provider has a specific URL path or not.
+  * Change `ICE_SERVER_API_KEY` to an API key or empty string depending if your ICE server provider requires an API key to access it or not.
+
+  ```python
+  ICE_SERVER_BASE_URL = 'https://networktraversal.googleapis.com'
+  ICE_SERVER_URL_TEMPLATE = '%s/v1alpha/iceconfig?key=%s'
+  ICE_SERVER_API_KEY = os.environ.get('ICE_SERVER_API_KEY')
+  ```
 
-8\. **(Running locally using the dev server (dev/testing purposes))** Build AppRTC using `grunt build` and then start it using dev appserver provided by GAE
-`pathToGAESDK/dev_appserver.py  out/app_engine/`.
+8\. Build AppRTC using `grunt build` then deploy/run:
+* **If Running locally using the appengine dev server (dev/testing purposes)**
+    * Start it using dev appserver provided by the Google app engine SDK `pathToGAESDK/dev_appserver.py  out/app_engine/`.
 
-9\. **(Running on Google App Engine (Production))**
-* Make sure you have a [Google cloud account and Google app engine enabled](https://cloud.google.com/appengine/docs/python/quickstart).
-* [Download the Google cloud SDK and initialize it](https://cloud.google.com/appengine/docs/python/tools/uploadinganapp).
-* Deploy your AppRTC app by executing the following the AppRTC checkout root directory `gcloud app deploy --project [YOUR_PROJECT_ID] -v [YOUR_VERSION_ID]` (You can find the [YOUR_PROJECT_ID] and [YOUR_VERSION_ID] in your Google cloud console).
+* **Else if running on Google App Engine in the cloud (Production)**
+  * Make sure you have a [Google cloud account and Google app engine enabled](https://cloud.google.com/appengine/docs/python/quickstart).
```

Comment:
Google cloud account -> Google Cloud Account since it's used as a product entity here.
Google app engine -> Google App Engine